decidim-core/spec/lib/commands/destroy_resource_spec.rb

```diff
@@ -2,11 +2,11 @@
 
 require "spec_helper"
 
-module Decidim::Accountability
-  describe Admin::DestroyResult do
+module Decidim
+  describe Commands::DestroyResource do
     subject { described_class.new(result, user) }
```

Comment:
Let's make it more generic 
```suggestion
    subject { described_class.new(resource, user) }
```